pkgs/by-name/cb/cbmp/default.nix

```
@@ -0,0 +1,35 @@
+{ buildNpmPackage, fetchFromGitHub, lib, ... }:
+buildNpmPackage rec {
+  pname = "cbmp";
+  version = "1.1.1";
+
+  # note: updating notes
+  # - use `prefetch-npm-deps` package for src hash
+  # - use `npm install --package-lock-only`
+  #   in the cbmp repo for package-lock generation
+  # - update npmDepsHash
+
+  src = fetchFromGitHub {
+    owner = "ful1e5";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-vOEz2KGJLCiiX+Or9y0JE9UF7sYbwaSCVm5iBv4jIdI=";
+  };
+
+  npmDepsHash = "sha256-zfZAZqnkKwsGkwGgfPvb3me4tFgAofE5H3d23mdAtqY=";
+
+  env = {
+    PUPPETEER_SKIP_DOWNLOAD = true;
+  };
+
+  postPatch = ''
+    cp ${./package-lock.json} package-lock.json
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/292468